### DISCUSSION TITLE
Github website trunkate some page.

### DISCUSSION THREAD

**Discussion by Smanar** [community/community#23254] [New to GitHub]
<p>Hello, I think a capture is more easy to understand</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/4f93a5e34fcace49531184b9855f05c783a1d44f.jpeg" data-download-href="/uploads/short-url/blY3nFQmTNy0vg8XaZoTyZMwjUj.jpg?dl=1" title="netu.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/20152487/181075939-33d456a2-0894-41f0-a870-2fa01a8af3f9.jpg" alt="netu.jpg" data-base62-sha1="blY3nFQmTNy0vg8XaZoTyZMwjUj" width="690" height="379" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/4f93a5e34fcace49531184b9855f05c783a1d44f_2_690x379.jpg, https://aws1.discourse-cdn.com/github/optimized/1X/4f93a5e34fcace49531184b9855f05c783a1d44f_2_1035x568.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/4f93a5e34fcace49531184b9855f05c783a1d44f_2_1380x758.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/4f93a5e34fcace49531184b9855f05c783a1d44f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">netu.jpg</span><span class="informations">1892×1041 143 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I realy don’t have idea what happen</p>
<p>For information.</p>
<ul>
<li>I have tried on different browser.</li>
<li>I have tried on different account.</li>
<li>It don’t appear on all pages, but the same page bug on all tests.</li>
<li>I have tried with plugin disabled.</li>
<li>I have take a look on the html code, the code is realy trunkated, it’s not a display bug</li>
</ul>
<p>But it work fine if I m not registerer. I have the bug only with registration (and different account).</p>
<p>Regards.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/smanar">@smanar</a>, Thanks for being here and for taking the time to report this. We are not experiencing any outages, if you are still experiencing this issue while using our <a href="https://help.github.com/en/github/getting-started-with-github/supported-browsers" rel="nofollow noopener">supported compatible browsers</a> please reach out to us at <a href="http://git.io/c" class="inline-onebox-loading" rel="nofollow noopener">git.io/c</a> and we’ll take a look.</p>

**Comment by Smanar** [comment, marked as answer]
<p>Hello, I have the bug for exemple on</p>
<p>Firefox 76.0.1</p>
<p>Chrome 81.0.4044.138</p>
<p>And yes, I m with the support ATM.</p>
<p>I seem I have the bug on big page, but If I m using TOR, it works even with registration, with the same browser.</p>
<p>Edit :</p>
<p>Hello, so good news, it works fine now, but bad news, I realy don’t understand what happened.<br>
Today just before answer to the support, I have reboot the machine, disable firewall/antivirus/applications, and I m re-enabling all the system part after part, and for the moment all is working.</p>
<p>I realy can’t say if the problem is from a security software or other thing blocked in memory, but 1h I m trying to reproduce the bug, without success.</p>
<p>I let the topic open for some days, time for me making some tries. But it’s realy a mystery for me, how it can work using the same browser (firefox), the same account, on TOR but not without, or why it work without registration and bug with …</p>
<p>Edit2:</p>
<p>Ok, so no more bug, and still unable to reproduce it.</p>
